src/App.js

```diff
@@ -9,6 +8,7 @@ import {
   MapManager,
   OurTable
 } from './components'
+import NavBar from './components/NavBar'
```

Comment:
If you change `components/index.js`, you can set it so that you are able to import from the directory like:

```jsx
import { NavBar } from './components'
```
Follow the pattern of importing and exporting that the other components use